components/Instructions/editRecipeInstructions.jsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from "react";
+
+function EditRecipeInstructions({ recipes }) {
+  const [instructions, setInstructions] = useState([]);
+
+  useEffect(() => {
+    if (recipes) {
+      const fetchRecipeInstructions = async () => {
+        const response = await fetch(`/api/recipes/${recipe.id}/instructions`);
+        const instructions = await response.json();
+
+        setInstructions(instructions);
+      };
+      fetchRecipeInstructions();
+    }
+  }, [recipes]);
+
+  const handleInstructionChange = (index, newInstruction) => {
+    setInstructions((prevInstructions) => {
+      const updatedInstructions = [...prevInstructions];
+      updatedInstructions[index] = newInstruction;
+      return updatedInstructions;
+    });
+  };
+
+  const handleSaveChanges = async () => {
```

Comment:
This can be a function in the db helpers.

Reply:
I am not the one working on it

Reply:
Will be looking into that when refactoring the code

Reply:
We'll look at this when we implement best practices.